pwndbg/heap/ptmalloc.py

```diff
@@ -658,17 +658,219 @@ def __init__(self) -> None:
         # ptmalloc cache for current thread
         self._thread_cache: gdb.Value = None
 
-    def largebin_size_range_from_index(self, index):
-        index += NSMALLBINS
-        spaces_table = self._spaces_table()
-        largest_largebin = self.largebin_index(pwndbg.gdblib.arch.ptrmask)
-        start_size = (NSMALLBINS * self.malloc_alignment) - self.malloc_alignment
+        self.largebin_reverse_lookup_32 = [
```

Comment:
Can we add some comments on those? E.g. where are those values taken from etc.

Reply:
I've added a comment briefly explaining how these tables were generated 👍